.golangci.yml

```diff
@@ -74,9 +74,7 @@ linters-settings:
 linters:
   enable-all: true
   disable:
-    - testpackage # TODO enable
     - goerr113 # TODO enable
-    - nlreturn # TODO enable
```

Comment:
I would in all seriousness forget about `goerr113` (see https://github.com/Djarvur/go-err113/issues/10).... or at most, leave a TODO to followup on closure of that issue and _maybe_ enable it again.

Reply:
Removed TODO.